server.js

```diff
@@ -13,8 +13,67 @@ app.get("/", function (request, response) {
   response.send("Hotel booking server.  Ask for /bookings, etc.");
 });
 
+app.get("/booking", function (req, res) {
+  res.send(bookings);
+});
+
+app.get("/booking/:id", function (req, res) {
+  const bookingID = Number(req.params.id);
+  const booking = bookings.find((booking) => booking.id === bookingID);
+  const bookingIndex = bookings.findIndex(
+    (booking) => booking.id === bookingID
+  );
+  if (bookingIndex === -1) {
```

Comment:
The `find` method will return undefined if it can't find the book, so you don't need to find the index again, you can just if `booking` is undefined